test/steps/general/3_then.js

```diff
@@ -35,13 +34,29 @@ export function itShouldReturnTheResult() {
 export function itShouldThrowValidationError() {
 	const { testFunction } = this.test.ctx;
 	const message = getFirstQuotedString(this.test.title);
-	return testFunction.should.throw(new ValidationError(message));
+	testFunction.should.throw().and.is.instanceOf(Error);
+	testFunction.should
+		.throw()
+		.and.has.property('name')
+		.which.equal('ValidationError');
+	return testFunction.should
+		.throw()
+		.and.has.property('message')
+		.which.include(message);
 }
 
 export function itShouldThrowFileSystemError() {
 	const { testFunction } = this.test.ctx;
 	const message = getFirstQuotedString(this.test.title);
-	return testFunction.should.throw(new FileSystemError(message));
+	testFunction.should.throw().and.is.instanceOf(Error);
+	testFunction.should
+		.throw()
+		.and.has.property('name')
+		.which.equal('FileSystemError');
+	return testFunction.should
+		.throw()
+		.and.has.property('message')
+		.which.include(message);
```

Comment:
Let's extract the common functionality into a helper function, so these `then` steps can be cleaner at least. Can we even add a method to chai?